its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/TelemetryTest.java

```diff
@@ -97,7 +97,7 @@ private void AssertTelemetry(String projectName) throws IOException {
 
     var sonarQubeOutDirectory = context.projectDir.resolve(".sonarqube").resolve("out");
 
-    assertThat(readContents(sonarQubeOutDirectory.resolve(("Telemetry.S4NET.json"))))
+    assertThat(readContents(sonarQubeOutDirectory.resolve(("rTelemetry.S4NET.json"))))
```

Comment:
Why is there the `r` letter?

Reply:
discussed offline:
r means the file has been processed by the C# or the VB.NET plugin. Otherwise in case of mixed projects we risk processing the same file twice.
There's a ticket already opened by Pavel to rename r -> processed.